### PR TITLE
Fix #324 - replace django-international with pycountry

### LINKED ISSUE #324: Find a replacement for django-international (no update in 2+ years)
Reason: https://github.com/PressLabs/silver/issues/322

Possible solutions:
https://pypi.python.org/pypi/pycountry
https://pypi.python.org/pypi/django-countries

### PR DESCRIPTION
**Pull request by Marius786**

### CODE REVIEW

File: silver/tests/factories.py [hunk 1]
```diff
@@ -17,17 +17,20 @@
...
 from django.contrib.auth import get_user_model
 from django.utils import timezone
-from international.models import countries
 
 from silver.models import (Provider, Plan, MeteredFeature, Customer,
                            Subscription, Invoice, ProductCode,
                            Proforma, MeteredFeatureUnitsLog, DocumentEntry)
 
 
+countries = [ (c.alpha2, c.name) for c in pycountry.countries ]
```
Comment (vtemian) [inline silver/tests/factories.py]: Be more explicit 
```[(country.alpha2, country.name) for country in pycountry.countries]```

File: silver/models.py [hunk 2]
```diff
@@ -49,6 +49,9 @@
 from silver.utils import next_month, prev_month
 from silver.validators import validate_reference
 
+countries = [ (c.alpha2, c.name) for c in pycountry.countries ]
```
Comment (vtemian) [inline silver/models.py]: Be more explicit 
```[(country.alpha2, country.name) for country in pycountry.countries]```

File: silver/tests/spec/test_invoice.py [hunk 3]
```diff
@@ -61,8 +61,8 @@ def test_post_invoice_without_invoice_entries(self):
             "number": None,
             "provider": "http://testserver/providers/1/",
             "customer": "http://testserver/customers/1/",
-            "archived_provider": {},
-            "archived_customer": {},
+            "archived_provider": None,
```
Comment (vtemian) [inline silver/tests/spec/test_invoice.py]: Why is there a `None` and not `{}`

Comment (Marius786) [inline silver/tests/spec/test_invoice.py]: Because the tests not passed. The returned values were with 'None' and not with '{}' see with drone exec.

**Comment by bogdanpetrea** [comment]
LGTM

**Pull request merged**